src/Symfony/Component/Console/Helper/ProcessHelper.php

```diff
@@ -23,6 +23,13 @@
  */
 class ProcessHelper extends Helper
 {
+    private $formatter;
+
+    public function __construct()
+    {
+        $this->formatter = new DebugFormatterHelper();
```

Comment:
what about an optional constructor arg for it? Assuming one wants to bridge a custom formatter ...

Reply:
Do you recommend a typehint? A new interface maybe for formatting helpers?

Reply:
@codedmonkey i was thinking of 

```
__construct(DebugFormatterHelper $formatter = null) 

$this->formatter = $formatter ?? new  DebugFormatterHelper();
```